cli/src/main/java/ai/wanaku/cli/main/support/FileHelper.java

```diff
@@ -0,0 +1,28 @@
+package ai.wanaku.cli.main.support;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class FileHelper {
+
+    private FileHelper() {}
+
+    public static boolean canWriteToDirectory(Path dir) {
+        if (dir == null) {
+            return false;
+        }
+
+        try {
+            Path tempFile = Files.createTempFile(dir, "test", ".tmp");
```

Comment:
Maybe let's pick a more appropriate prefix here. Let's say ... `wanaku-tmp` instead of `test`.